src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -164,27 +171,43 @@ public class CoinstakeContext
         /// but high enough to compensate for tasks' overhead.</remarks>
         private const int UtxoStakeDescriptionsPerCoinstakeWorker = 25;
 
+        /// <summary>Responsible for downloading and validating blocks.</summary>
         private readonly ConsensusLoop consensusLoop;
+
+        /// <summary>Chain of headers from genesis.</summary>
         private readonly ConcurrentChain chain;
 
         /// <summary>Specification of the network the node runs on - regtest/testnet/mainnet.</summary>
         private readonly Network network;
+
+        /// <summary>Used to verify that node is connected to network before we start staking.</summary>
```

Comment:
implementation -> this could go to remarks
in summary, i'd prefer something like `Provider of information about the node's connection to its network peers.`

Reply:
as mentioned on slack, all those "Used to" and alike, i'm not a big fan of